charts/Pie.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {argTypes, Pie} from './Pie.args';
+import configify from "../helpers/configify";
+
+export default {
+  title: "Charts/Pie",
+  component: Pie,
+  argTypes
+};
+
+const Template = (args) => <Pie config={configify(args)} />;
+
+export const GettingStarted = Template.bind({});
+GettingStarted.args = {
+  data: [
+    {id: "White Health Center Patients", value: 40},
+    {id: "Black Health Center Patients", value: 20},
+    {id: "Hispanic Health Center Patients", value: 25},
+    {id: "Asian Health Center Patients", value: 10},
+    {id: "American Indian Health Center Patients", value: 5}
+  ],
+  groupBy: "id",
+  value: "value"
```

Comment:
Let's start using more descriptive keys in our examples. For this one, I would say "Demographic" instead of "id" and "Percentage" instead of "value".

Reply:
I changed this example to pie types and also modified the `groupBy` and `value` accessor names 🥧 
```
  data: [
    {Type: "Apple pie", Percentage: 30},
    {Type: "Butter pie", Percentage: 20},
    {Type: "Cherry pie", Percentage: 25},
    {Type: "Lemon pie", Percentage: 15},
    {Type: "Suggar pie", Percentage: 10}
  ],
  groupBy: "Type",
  value: "Percentage"
```